### PR TITLE
progressbar in menu

### PR DESCRIPTION
**Pull request by kewde**
- progress bar in menu
- test exponential moving average for time left
- get rid of blocks state usage in blockstatus

### CODE REVIEW

File: src/app/core/rpc/peer.service.ts [hunk 1]
```diff
@@ -51,6 +54,11 @@ export class PeerService {
       error => this.log.er(`updatePeerListLoop(): getpeerinfo error ${error}`)
     );
 
+    this._rpc.call('getblockcount').subscribe(
```
Comment (rynomster) [inline src/app/core/rpc/peer.service.ts]: Why would you do this?

Comment (rynomster) [reply src/app/core/rpc/peer.service.ts]: We already have it in state..

Comment (kewde) [reply src/app/core/rpc/peer.service.ts]: It caused a hanging in the last few blocks, it would keep the sync modal open longer than required

**Comment by kewde** [comment]
Still needs GFX fixed.

**Comment by allienworks** [comment]
I've styled the progress bar, here's a [preview for the lazy](https://imgur.com/a/443Pm) – @gerlofvanek approved? :)
It's visible while it's syncing, disappears when at 100%.

Also couldn't help not to tweak the logo & sidebar a bit.

**Comment by allienworks** [comment]
@rynomster can we merge?

We need this merged so that https://github.com/particl/partgui/pull/342 can follow.

**Pull request merged**